node/node_test.go

```diff
@@ -75,3 +77,17 @@ func TestSplitAndTrimEmpty(t *testing.T) {
 		assert.Equal(t, tc.expected, splitAndTrimEmpty(tc.s, tc.sep, tc.cutset), "%s", tc.s)
 	}
 }
+
+func TestNodeDelayedStop(t *testing.T) {
+	config := cfg.ResetTestRoot("node_delayed_node_test")
+	now := tmtime.Now()
+
+	// create & start node
+	n, err := DefaultNewNode(config, log.TestingLogger())
+	n.GenesisDoc().GenesisTime = now.Add(5 * time.Second)
```

Comment:
Does this reliably pass for smaller values?

Reply:
It pass for 10 millisecond.